### PR TITLE
57 create script to automate env creation

### PR DESCRIPTION
**Pull request by tsiguenz**

### CODE REVIEW

File: set_env_file.sh [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  echo_colored "${LIGHT_GREEN}" "     /      |      \\ "
+  echo_colored "${LIGHT_GREEN}" "    #################"
+  echo_colored "${LIGHT_GREEN}" "   /   _ ( )|        \\ "
+  echo_colored "${LIGHT_GREEN}" "  /   ( ) | |         \\ "
+  echo_colored "${LIGHT_GREEN}" " /  \  |_/  |          \\ "
+  echo_colored "${LIGHT_GREEN}" "/____\/|____|___________\\ "
+  echo_colored "${LIGHT_GREEN}" "   |   |             |"
+  echo_colored "${LIGHT_GREEN}" "   |  / \            |"
+  echo_colored "${LIGHT_GREEN}" "   | /   \           |"
+  echo_colored "${LIGHT_GREEN}" "   _/    /_\n"
```
Comment (Crysicia) [inline set_env_file.sh:28]: Most important part of the script, GG

**Review by Crysicia — APPROVED** [review]
LGTM, congrats on doing bash for more than 2 minutes without getting depression

File: check_env_file.sh [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  local AUTHORIZE_URL=https://api.intra.42.fr/oauth/authorize
+  local TOKEN_URL=https://api.intra.42.fr/oauth/token
+
+  local APP42_UID=$(grep APP42_ID ${ENV_FILE} | cut -d '"' -f2)
+  local APP42_SECRET=$(grep APP42_KEY ${ENV_FILE} | cut -d '"' -f2)
+  local APP42_REDIRECT_URI=http://localhost:8080/42/callback
+
+  echo_colored ${CYAN} "Getting credentials..."
+  echo_colored ${CYAN} "Redirecting to 42's authorization page..."
+  open "${AUTHORIZE_URL}?client_id=${APP42_UID}&redirect_uri=http://localhost:8080/42/callback&response_type=code" > /dev/null 2>&1
```
Comment (gmorange42) [inline check_env_file.sh:27]: Use redirect variable.

File: check_env_file.sh [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  echo_colored ${CYAN} "Please enter the authorization code:"
+  read AUTHORIZATION_CODE
+  echo_colored ${CYAN} "Getting access token..."
+  local ACCESS_TOKEN=$(curl -F grant_type=authorization_code \
+    -F client_id=${APP42_UID} \
+    -F client_secret=${APP42_SECRET} \
+    -F code=${AUTHORIZATION_CODE} \
+    -F redirect_uri=${APP42_REDIRECT_URI} \
+    -X POST https://api.intra.42.fr/oauth/token \
```
Comment (gmorange42) [inline check_env_file.sh:36]: Use token variable.

**Review by gmorange42 — APPROVED** [review]
Nice job, i'll never ask you "Why it doesn't works?"

**Pull request merged**